doc/release-notes/0.40/0.40.md

```diff
@@ -0,0 +1,122 @@
+<td valign="top">Persistent shared caches are now supported on z/OS systems. With the 31-bit VM on all versions of z/OS, the shared cache is memory mapped within the 0-2 GB address range. With the 64-bit VM on z/OS version 2.3 and earlier, the persistent shared cache is memory mapped within the 0-2 GB address range but on z/OS version 2.4 and later the shared cache is memory mapped above the 2 GB address range. </td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17425">#17425</a></td>
+<td valign="top">A new <tt>-XX:codecachetotalMaxRAMPercentage</tt> option is added to set the maximum limit for the total JIT code cache size based on the amount of physical memory available to the VM.</td>
+<td valign="top">All versions</td>
+<td valign="top">The new option is used to set the code cache maximum size limit as a percentage of the available physical memory. In memory constrained environments, the percentage limit prevents the JIT from using too much memory for its code caches and thus, leaving too little memory for other VM or JIT data structures. By default, the VM is not allowed to use more than 25% of the available physical memory for its code caches.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse/omr/issues/7021">#7021</a></td>
+<td valign="top">When <tt>/proc</tt> is mounted with <tt>hidepid=2</tt> on Linux&reg; systems, the container detection fails but the VM does not exit. Now, the VM starts but with a warning message.</td>
+<td valign="top">All versions (Linux)</td>
+<td valign="top">Although the VM starts after the container detection fails, the VM assumes that it is running outside a container. Therefore, if the VM is running in a container, the VM cannot adapt to the container's limitations and might use an undesirable amount of resources. You can evaluate the impact on performance because of the container detection failure and take steps to resolve the performance issue, if so required. For example, you can remount the <tt>/proc</tt> file system with the <tt>gid</tt> and <tt>hidepid=2</tt> options to allow some processes to access the <tt>/proc</tt> file system. </td>
+</tr>
+
+</tbody>
+</table>
+
+## Known issues
+
+The v0.40.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr>
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+
+<tbody>
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15011">#15011</a></td>
+<td valign="top">The default stack size is modified for the main thread.</td>
+<td valign="top">All</td>
+<td valign="top">The main thread stack size was 1 MB in releases prior to 0.32. In the 0.32 release and later it's modified to a smaller
```

Comment:
As this happened quite a while ago, perhaps "it's" should be "it was"?  Similarly below, "The 0.33 release increases" should probably be "The 0.33 release increased".